contribution-guide.md

```diff
@@ -0,0 +1,154 @@
+- [Set up the Prerequisites](#set-up-the-prerequisites)
+- [Set up the developer environment](#set-up-the-developer-environment)
+    - [Set up IntelliJ IDEA](#set-up-intellij-idea)
+    - [Set up Eclipse](#set-up-eclipse)
+- [Obtain the source code](#obtain-the-source-code)
+- [Build the project](#build-the-project)
+- [Contribute to Ballerina grammar](#contribute-to-ballerina-grammar)
+- [Submit your contribution](#submit-your-contribution)
+- [Accept the Contributor License Agreement](#accept-the-contributor-license-agreement)
+- [Propose changes to Ballerina](#propose-changes-to-ballerina)
+
+## Get started
+
+- Join the conversations at:
+
+    - [Ballerina-Dev Google Group](https://groups.google.com/forum/#!forum/ballerina-dev): to join the developer team mailing list to discuss Ballerina roadmap, features, issues that are in progress etc.
+    - [StackOverflow](https://stackoverflow.com/questions/tagged/ballerina): to get help with Ballerina. (Use the Ballerina tag for your questions there.) 
+    - [Slack](https://app.slack.com/client/T47EAEKB5/DF818FX9Q): for real-time communications with the team and community.
+
+- **IMPORTANT:** Send an email to security@ballerina.io to report sensitive security-related issues. For details, see the <a href="https://ballerina.io/security/">security policy</a>.
+
+- Submitting a bug is just as important as contributing to code. Create a new issue in the relevant GitHub repo using the below links, fill out all sections in the issue template, and submit the bug/issue report. 
+
+    >**Tip:** If you are unsure whether you have found a bug, search existing issues in the corresponding repo on GitHub and raise it in the [Ballerina-Dev Google Group](#https://groups.google.com/forum/#!forum/ballerina-dev).
+   - compiler, runtime, standard library, or tooling: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
+   - language specification: <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo.
+   - website: <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo. 
+
+-  Start with GitHub issues that can be fixed easily:
+    - Browse issues labeled "good first issue" in the <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
+    - Use comments on the issue itself to indicate that you will be working on it and get guidance and help.
+
+## Set up the prerequisites
+1. Download [Ballerina](https://ballerina.io) and go through the [learning resources](https://ballerina.io/learn).
+2. Read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
+3. Download the Java SE Development Kit (JDK) version 8 from one of the following locations:
+    - [Oracle](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+    - [OpenJDK](http://openjdk.java.net/install/index.html) and install it.
```

Comment:
It seems the "and install it" should not be part of the bullet point as it is common for both points. Shall we change the main line to the following?

"Download the Java SE Development Kit (JDK) version 8 from one of the following locations and install it."

Reply:
Done